tests/marqo_test.py

```diff
@@ -105,6 +105,16 @@ def wrapper(self, *args, **kwargs):
     return decorator
 
 
+def create_settings_hash(settings_dict, kwargs):
```

Comment:
create a doc string about this.
- explain the purpose of settings_hash
- how it fits into the wider context of the cloud test architecture 
- Explain choices, such as why the hash is truncated at 10 chars